src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/VisitMappingResourceIntTest.kt

```diff
@@ -190,6 +190,53 @@ class MappingResourceIntTest : IntegrationTestBase() {
       assertThat(mapping2.label).isEqualTo("2022-01-01")
       assertThat(mapping2.mappingType).isEqualTo("ONLINE")
     }
+
+    @Nested
+    @DisplayName("/mapping/visits")
+    inner class VisitPath {
+      @Test
+      fun `create mapping success`() {
```

Comment:
same test is repeated but with new path; when old path is removed all tests will be amended and the duplicate ones will be removed